client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx

```diff
@@ -200,6 +204,8 @@ const DomainsStep: Step = function DomainsStep( { navigation, flow } ) {
 					),
 					decideLaterComponent
 				);
+			case COPY_SITE_FLOW:
+				return __( 'Set your Link in Bio apart with a custom domain.' );
```

Comment:
Let's use something like this:

```
Help your freshly copied site stand out with a custom domain. Not sure yet? <span>Decide later</span>.
```

Reply:
I thnk that maybe we should remove the `<span>Decide later</span>.` as it would give the user the ability to skip

Reply:
@kozer Ok, that sounds fine.